packages/app/src/app/pages/Dashboard/Content/routes/Repositories/EmptyRepositories.tsx

```diff
@@ -57,6 +85,233 @@ export const EmptyRepositories: React.FC = () => {
+  const { isTeamSpace } = useWorkspaceAuthorization();
+  const { isFree } = useWorkspaceSubscription();
+
+  const selectOptions = useMemo(
+    () =>
+      githubAccounts.data && githubAccounts.data.personal
+        ? [
+            githubAccounts.data.personal,
+            ...(githubAccounts.data.organizations || []),
+          ]
+        : undefined,
+    [githubAccounts.data]
+  );
+
+  useEffect(() => {
+    // Set initially selected account bazed on fuzzy matching
+    if (
+      githubAccounts.state === 'ready' &&
+      // Adding selectOptions to this if statement to satisfy TypeScript, because it does not
+      // know that when githubAccounts.state !== 'ready' the fuzzy functions isn't executed.
+      selectOptions &&
+      activeTeamInfo?.name &&
+      selectedAccount === undefined
+    ) {
+      const match = fuzzyMatchGithubToCsb(activeTeamInfo.name, selectOptions);
+
+      setSelectedAccount(match.login);
+    }
+  }, [githubAccounts.state, selectedAccount, activeTeamInfo, selectOptions]);
+
+  // eslint-disable-next-line no-nested-ternary
+  const selectedAccountType = selectedAccount
+    ? selectedAccount === githubAccounts?.data?.personal?.login
+      ? 'personal'
+      : 'organization'
+    : undefined;
+
+  const githubRepos = useGitHubAccountRepositories({
+    name: selectedAccount,
+    accountType: selectedAccountType,
+  });
+
+  const handleImport = async (
+    repo: Pick<Repository['repository'], 'owner' | 'name'>
+  ) => {
+    setIsImporting(true);
+
+    const isPersonalRepository =
+      repo.owner === githubAccounts?.data?.personal?.login;
+
+    if (isPersonalRepository && isTeamSpace) {
+      track(
+        'Suggested repos empty page - Imported personal repository into team space',
+        {
+          codesandbox: 'V1',
+          event_source: 'UI',
+        }
+      );
+    } else {
+      track(
+        'Suggested repos empty page - Imported organization repository into team space',
+        {
+          codesandbox: 'V1',
+          event_source: 'UI',
+        }
+      );
+    }
+
+    await dashboardActions.importGitHubRepository(repo);
+  };
+
+  return (
+    <>
+      <Stack gap={2}>
+        <Text size={16}>Import from </Text>
+        {githubAccounts.state === 'loading' ? (
+          <SkeletonText />
+        ) : (
+          <AccountSelect
+            options={selectOptions}
+            value={selectedAccount}
+            onChange={(account: string) => {
+              track('Suggested repos - change account', {
+                codesandbox: 'V1',
+                event_source: 'UI',
+              });
+
+              setSelectedAccount(account);
+            }}
+          />
+        )}
+      </Stack>
+
+      {githubRepos.state === 'loading' ? (
+        <EmptyPage.StyledGrid>
+          <SolidSkeleton viewMode="grid" />
+          <SolidSkeleton viewMode="grid" />
+          <SolidSkeleton viewMode="grid" />
+        </EmptyPage.StyledGrid>
```

Comment:
What do you think about making the number of skeletons the same as the number of cards we intend to show?